linux/runtime.go

```diff
@@ -204,6 +207,7 @@ func (r *Runtime) forward(events shim.Shim_EventsClient) {
 			Pid:        e.Pid,
 			ID:         e.ID,
 			ExitStatus: e.ExitStatus,
+			ExitedAt:   e.ExitedAt,
```

Comment:
Why do we always set event timestamp in containerd? Why don't we generate timestamp in the shim?

Just curious. :)